scipy/stats/meson.build

```diff
@@ -119,7 +119,8 @@ _stats_gen_pyx = custom_target('_stats_gen_pyx',
     'hypergeom_ufunc.pyx',  # 4 (S_B)
     'nbinom_ufunc.pyx',  # 5 (S_B)
     'templated_pyufunc.pxd',  # 6 (S_B)
-    'unuran_wrapper.pyx'  # 7 (used in stats/_unuran)
+    'unuran_wrapper.pyx',  # 7 (used in stats/_unuran)
+    'ncf_ufunc.pyx'  # 8 (S_B)
```

Comment:
Add the new generated pyx file to the list of Cython files that meson tracks -- make note of the index in this list, it will be used below